packages/app/src/app/pages/Dashboard/Header/index.tsx

```diff
@@ -121,6 +121,22 @@ export const Header: React.FC<HeaderProps> = React.memo(
             Create
           </Button>
 
+          <Button
+            variant="ghost"
+            css={css({ width: 'auto' })}
+            onClick={() => {
+              window.open('http://codesandbox.io/discover', '_blank');
+            }}
+          >
+            <Icon
+              name="discover"
+              size={22}
+              title="New"
+              css={css({ paddingRight: 2 })}
```

Comment:
```suggestion
              css={{ paddingRight: 2 }}
```

We've dropped the `css` from new code to make migration to the v2 codebase easier.